easybuild/easyblocks/m/mvapich2.py

```diff
@@ -67,7 +67,11 @@ def configure_step(self):
 
         # additional configuration options
         add_configopts = []
-        add_configopts.append('--with-rdma=%s' % self.cfg['rdma_type'])
+
+        if self.cfg['rdma_type']:
+            add_configopts.append('--with-rdma=%s' % self.cfg['rdma_type'])
+        else:
+            add_configopts.append('--without-rdma')
```

Comment:
Hmm, when testing this with the existing `MVAPICH2-2.0.1-GCC-4.8.4.eb`, I ran into this:

```
configure: error: The specified RDMA type is not supported
```

@peterinl: Maybe `--without-rdma` is not sufficient? Should this be something different?